modules/frontend/frontend-js-web/src/main/resources/META-INF/resources/liferay/layouts_tree.js

```diff
@@ -7,7 +7,7 @@ AUI.add(
 
 		var NODE_ID_TPL = '{treeId}_layoutId_{layoutId}_plid_{plid}_groupId_{groupId}';
 
-		var NODE_LINK_TPL = '<a class="{cssClass}" data-url="{url}" data-uuid="{uuid}" href="{layoutURL}" id="{id}" title="{title}">{label}</a>';
+		var NODE_LINK_TPL = '<a class="{cssClass}" data-navigation="{navigation}" data-url="{url}" data-uuid="{uuid}" href="{layoutURL}" id="{id}" title="{title}">{label}</a>';
```

Comment:
@juliocamarero this will cause all links in the tree to disable SPA. based on the code below, I assume that you want spa to affect only one of the layouts set. You should remove the `data-navigation` attribute alltogether in the one you'd want SPA enabled.